dmcontent/govuk_frontend.py

```diff
@@ -107,11 +107,30 @@ def from_question(
 def govuk_input(
     question: Question, data: Optional[dict] = None, errors: Optional[dict] = None, **kwargs
 ) -> dict:
-    """Create govukInput macro parameters from a text question"""
+    """Create govukInput macro parameters from a text or number question"""
 
     params = _params(question, data, errors)
     params["classes"] = "app-text-input--height-compatible"
 
+    if question.type == "number":
+        params["classes"] += " govuk-input--width-5"
```

Comment:
I'm guessing this is all that's used on Briefs? (ie: we don't have to allow for different widths right now unless they pop up on other apps?).

Reply:
Yeah, the only places we use it the size is currently just `smaller` 😆 https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/forms/macros/forms.html#L232